test/house_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'minitest'
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def test_it_exists
+    house = House.new('$400000', '123 sugar lane', [])
+    assert_instance_of House, house
+  end
+
+  def test_it_has_price_address_rooms
+    assert_equal '$400000', @house.price
```

Comment:
So the reason that you were seeing `NoMethodError: undefined method price' for nil:NilClass test/house_test.rb:15:in ` is because `@house` was never defined in your test. For each method in your test you need to create a new instance of `House` just like you did on line 10 of this file in `test_it_exists`. That should be the starting point of every test method. Let me know if you have more questions about this.